recipes/ganon/meta.yaml

```diff
@@ -30,26 +30,28 @@ requirements:
     - catch2 ==2.*
     - cxxopts >=2.2.0
     - python
-    - pandas >=1.1.0
+    - pandas >=1.2.0
     - multitax >=1.3.1
     - genome_updater >=0.6.3
     - grep
     - coreutils
     - curl
     - diffutils
     - zlib
+    - parameterized >=0.9.0
+    - raptor ==3.0.1=h6dccd9a_2
```

Comment:
Why is this strict pinning needed?

Reply:
This pinning is based on the PR #44952 adding multiple binaries depending on the instruction sets for. I believe the original raptor build v3.0.1 was forcing `-avx2` and wouldn't  run on CPUs without AVX, as reported by some users.

Reply:
Shouldn't we retract the broken version then?

Reply:
Yes, I think so. How can I retract broken builds?

Reply:
Can you tell me the exact name of the package(s) on [anaconda](https://anaconda.org/bioconda/raptor)?

Reply:
Those 2 builds can be retracted, since both break in case of missing instruction sets:
[linux-64/raptor-3.0.1-h6dccd9a_0.tar.bz2](https://anaconda.org/bioconda/raptor/3.0.1/download/linux-64/raptor-3.0.1-h6dccd9a_0.tar.bz2)
[linux-64/raptor-3.0.1-h6dccd9a_1.tar.bz2](https://anaconda.org/bioconda/raptor/3.0.1/download/linux-64/raptor-3.0.1-h6dccd9a_1.tar.bz2)

Reply:
Ok, they are now gone from the main index. I think you can remove this pin.

Reply:
@bgruening Pin removed, thanks! if ready, could you please merge?